mmv1/templates/terraform/examples/firebase_extentions_instance_resize_image.tf.erb

```diff
@@ -33,7 +33,6 @@ resource "google_firebase_extensions_instance" "resize_image" {
 
     system_params = {
       "firebaseextensions.v1beta.function/maxInstances"               = 3000
-      "firebaseextensions.v1beta.function/memory"                     = 256
```

Comment:
quick question regarding this change - does it mean that `firebaseextensions.v1beta.function/memory` is not a valid input for system_params?

Reply:
`firebaseextensions.v1beta.function/memory` is a valid parameter according to https://firebase.google.com/docs/extensions/publishers/parameters#system_parameters. `FUNCTION_MEMORY` is a required parameter for this particular extension (storage-resize-image). I don't remember why I put both FUNCTION_MEMORY and "firebaseextensions.v1beta.function/memory" when I first made the PR. Maybe I copied from a different extension. However, it looks like the backend has changed so that it only sees FUNCTION_MEMORY now.